kuma/search/managers.py

```diff
@@ -12,6 +12,10 @@ class FilterManager(models.Manager):
     def visible_only(self):
         return self.filter(visible=True)
 
+    @lru_cache(maxsize=None)
```

Comment:
The "size" in `maxsize` is a bit of a misnomer. It's actually number of executions before it refreshes the cache. I'll update it to use something like 1024 or something.